R/R-RNetCDF/Portfile

```diff
@@ -3,8 +3,10 @@
 PortSystem          1.0
 PortGroup           R 1.0
 
+# Only works on macos-14 and later after g95 fix: https://trac.macports.org/ticket/71068
```

Comment:
@remkos I do not understand what you mean. It should work on macOS < 14, unless somebody has broken that.